html/infrastructure/safe-passing-of-structured-data/shared-array-buffers/data-url-agent-cluster.https.html

```diff
@@ -0,0 +1,45 @@
+<!doctype html>
+<meta charset=utf-8>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/wasm/jsapi/wasm-module-builder.js"></script>
+<div id=log></div>
+<script>
+const dataURLWorker = `
+  onmessage = () => self.postMessage("FAIL");
+  onmessageerror = () => self.postMessage("PASS");
+`;
+
+const dataFrameScript = `
+  onmessage = () => parent.postMessage("FAIL", "*");
+  onmessageerror = () => parent.postMessage("FAIL", "*");
+`;
```

Comment:
I would find the test easier to follow if each event posted its own name to the parent (e.g. `self.postMessage("message")`, `self.postMessage("messageerror")` and then the asserts tested against that. Then it's easier when reading the assert to see "ah, a messageerror is what is desired in this test case".